document/CoreFormat.mdk

```diff
@@ -986,29 +1094,80 @@ A `location` object may contain a property named `tags`, whose value is a tag se
 A `location` object may contain a property named `properties`, whose value is a property bag.
 This allows tools to include information about the location that is not explicitly specified in the SARIF format.
 
-## `physicalLocationComponent` object { #physicalLocationComponent }
+## `physicalLocation` object { #physicalLocation }
 
-### General { #physicalLocationComponent-General }
+### General { #physicalLocation-General }
 
-A `physicalLocationComponent` object is an element of a `physicalLocation` object (&sect;[#physicalLocation-info]).
+A `physicalLocation` object represents the physical location where a result was detected.
+A physical location specifies a reference to a programming artifact together with a range of bytes or characters within that artifact.
 
-### `uri` property { #physicalLocationComponent-uri }
+### `uri` property { #physicalLocation-uri }
 
-A `physicalLocationComponent` object shall contain a property named `uri` whose value is a string that represents
-the location of the file to be modified as a valid URI.
+A `physicalLocation` object shall contain a property named `uri` whose value is a string that represents
+the location of the file as a valid URI.
 
-### `mimeType` property { #physicalLocationComponent-mimeType }
+The value of the `uri` property should be equal to the name of one of the properties on the
+`runInfo.fileInfo` object (&sect;[#runInfo-fileInfo]).
+`fileInfo` provides additional information about the file specified by `uri`.
 
-A `physicalLocationComponent` object should contain a property named `mimeType` with a string value that specifies
-the MIME content-type (RFC 2045) of the file.
+~ IsoExample
+
+```
+{
+  "version": "1.0",
+  "runLogs": [
+    {
+      "runInfo": {
+        "fileInfo": {
+          "file:///C:/Code/main.cc": [
+            {
+              "mimeType": "text/x-c",
+              "tags": [ "entryPoint" ]
+            }
+          ]
+        }
+      },
+      
+      "results": [
+        {
+          "ruleId": "CA2101",
+          "kind": "error",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "file:///C:/Code/main.cc",
+                "region: {
+                  "startLine": 24,
+                  "startColumn": 9
+                }
+              },
+              "logicalLocation": [
```

Comment:
logicalLocations not strictly relevant to this example but inclusion here troubles me less than tags... :)